### ISSUE TITLE
Change grade system for Sport climbing for United Kingdom to French

### ISSUE THREAD

**Issue by longb1** [theCrag/website#4200]
Can the grade context be changed from UK to French for United Kingdom?

I'm assuming UK refers to Technical UK Grading which really only applies to Trad climbing (E3 5c, etc..), and most routes here are only applied the french grading (6a, etc..) so it doesn't really make sense, and creates confusion.

Thanks

**Comment by rouletout** [comment]
I'm not sure I understand - my interpretation of this is that the UK context stays but that for gear style sport the grading system is changed to FR whereas for Trad it stays UK. Is this correct? What about boulder then?

**Comment by longb1** [comment]
@rouletout yes correct, as for Boulder, i think french grade context automatically includes boulder aswell, for example:
setting french grade context on sport, it converts to : 6b
setting french grade context on boulder, it converts to : 6B

see boulder and top rope examples here: https://www.thecrag.com/en/climbing/united-kingdom/tunbridge-wells/harrisons-rocks/area/253044237

i know there is a grade context setting for {FB} - fontainbleau aswell, but this doesn't seem as frequently used as french {FR} for both sport and boulder.

If I'm wrong on this then it seems weird for French to capitalise 6b to 6B for boulders, and this also creates a point of confusion for people who add new routes (for example, see boulders here with varying grade contexts: https://www.thecrag.com/en/climbing/united-kingdom/cratcliffe-tor/area/3256453368)

anyways, back to the point, UK context is really only used for Trad, and setting all of the united kingdom to this grade context kinda causes confusion when people add new routes.

FOr example, see this crag here, where all routes are graded in the UK context
https://www.thecrag.com/en/climbing/united-kingdom/tunbridge-wells/unclimbed-wall
![image](https://github.com/theCrag/website/assets/17272827/1e251ab3-fa7e-4dd1-8048-06307607c8b2)

and see the grade for the corresponding routes in UKC (uk e-climbing guide):
![image](https://github.com/theCrag/website/assets/17272827/347cef04-a545-43be-a760-a2e9328f6311)

Even UKC, which is basically theCrag in the UK, is using French grade context for their routes

this causes confusion among users, see:
![image](https://github.com/theCrag/website/assets/17272827/6af5b6c3-edea-48fd-8276-ab3a93465ab1)

**Comment by rouletout** [comment]
I think there is a terminology issue here - the context says which grade system is applied for which gear style. So if I understand you right you want to change the grade system for Sport Climbing in the UK Context to be changed to French, correct? Read this for more info on contexts: https://www.thecrag.com/en/article/gradesonthecrag#article.gradesonthecrag.header.grade-contexts

**Comment by longb1** [comment]
ah - i see, i misunderstood grade context then, yes that's what i would like thanks

EDIT: can UK grade context mirror FR grade context for both sport climbing and boulder? and top rope...
